src/app/(project)/projects/[project]/drops/[drop]/holders/Holders.tsx

```diff
@@ -125,17 +125,31 @@ export default function Holders({ project, drop, loading }: HoldersProps) {
                 const address = info.row.original.address;
                 const options = [];
 
-                if (holdersQuery.data?.project.drop?.collection.blockchain === Blockchain.Solana) {
-                  options.push(
-                    <Link
-                      href={`https://solscan.io/account/${address}`}
-                      target="_blank"
-                      key="change_email"
-                      className="flex gap-2 items-center"
-                    >
-                      <Icon.ExternalLink /> <span>View on SolScan</span>
-                    </Link>
-                  );
+                switch (holdersQuery.data?.project.drop?.collection.blockchain) {
+                  case Blockchain.Solana:
+                    options.push(
+                      <Link
+                        href={`https://solscan.io/account/${address}`}
+                        target="_blank"
+                        key="solana"
+                        className="flex gap-2 items-center"
+                      >
+                        <Icon.ExternalLink /> <span>View on SolScan</span>
+                      </Link>
+                    );
+                    break;
+                  case Blockchain.Polygon:
+                    options.push(
+                      <Link
+                        href={`https://polygonscan.com/address/${address}`}
```

Comment:
The address is `{contractAddress}:{tokenId}` so we need to replace the `:` with a `/` and the url is 

https://polygonscan.com/nft/{contractAddress}/{tokenId}

This is a good candidate for a client field on the object of exploreLink that handles figuring encapsulates the link assembly logic.

And then have the label be `View on explorer` for all of them.